src/App.vue

```diff
@@ -5,6 +5,7 @@
     <template v-if="repos">
       <h1 v-if="!tie">{{ vueHasPassedReact ? 'YES' : 'NO' }}</h1>
       <h1 v-else>TIE!</h1>
+      <p class="left" v-if="!vueHasPassedReact">left: {{ left }}</p>
```

Comment:
I don't think think this is needed? It already says how many stars away.

Reply:
Really IE? replacing axios by native fetch will give us one more dependence less

Reply:
I think it would be nice to see how many stars still remain "for pass"